examples/openvino/audio-classification/requirements.txt

```diff
@@ -0,0 +1,4 @@
+datasets>=1.14.0
```

Comment:
Do we need NNCF installed from source here?

Reply:
> Do we need NNCF installed from source here?

Not needed.  JPQD is supported in NNCF 2.4 release, which is already in `setup.py` requirement.